### ISSUE TITLE
Force HTTPS link

### ISSUE THREAD

**Issue by csianglim** [UBCEnvision/UBCEnvision.github.io#265]
Automatically redirect to https://www.ubcenvision.com instead of http as per earlier discussion.

- Update settings in CloudFlare

**Comment by jd4900** [comment]
@victorchiew you have access to the website on CloudFlare right?

**Comment by csianglim** [comment]
@victorchiew @jd4900 further reading (HTTPS errors section): https://help.github.com/articles/troubleshooting-custom-domains/#https-errors

**Comment by csianglim** [comment]
@jd4900 link sent via dm

**Comment by jd4900** [comment]
Who has access to the DNS? Still need to force HTTPS. Account credentials should be passed to @JayTayah

**Comment by csianglim** [comment]
![image](https://user-images.githubusercontent.com/7519832/99476419-64580f00-2905-11eb-9b15-07fa6602c353.png)

^ August 2018 @jd4900

**Comment by victorchiew** [comment]
> @victorchiew you have access to the website on CloudFlare right?

Yup, just logged in to make sure, let me know which account I should reroute it to

**Comment by jd4900** [comment]
Send to admin@ubcenvision.com
